chatsky/responses/rag.py

```diff
@@ -0,0 +1,62 @@
+from typing import Dict, Any
+from sentence_transformers import SentenceTransformer
+from qdrant_client import QdrantClient
+from chatsky.responses.base_response import BaseResponse
```

Comment:
There should be an empty line separating import groups ([pep-8](https://peps.python.org/pep-0008/#imports)):
> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
> 
> Imports should be grouped in the following order:
> 
>     Standard library imports.
>     Related third party imports.
>     Local application/library specific imports.
> 
> You should put a blank line between each group of imports.
```python
from typing import Dict, Any

from sentence_transformers import SentenceTransformer
from qdrant_client import QdrantClient

from chatsky.responses.base_response import BaseResponse
```